test/WebSites/ModelBindingWebSite/Controllers/VehicleController.cs

```diff
@@ -13,6 +14,21 @@ namespace ModelBindingWebSite
 {
     public class VehicleController : Controller
     {
+        private static Dictionary<int, VehicleViewModel> _vehicles = new Dictionary<int, VehicleViewModel>
+        {
+            {
+                42,
+                new VehicleViewModel
+                {
+                    InspectedDates = new[] { DateTimeOffset.Parse("1 April 2001"), },
```

Comment:
functional tests run sites using `CultureReplacerMiddleware` and that guarantees en-GB date/time formatting.  but I switched to the `DateTimeOffset` constructor since we aren't parsing `DateTime` or `DateTimeOffset` strings in other tests.